packages/train2go-bridge/popup.js

```
@@ -0,0 +1,65 @@
+};
+
+$("check-btn").addEventListener("click", check);
+
+$("read-btn").addEventListener("click", () => {
+  if (!storedUserId) return;
+
+  const today = new Date().toISOString().split("T")[0];
+  $("activities").textContent = "Loading...";
+  $("read-btn").disabled = true;
+
+  chrome.runtime.sendMessage(
```

ESLint

[error] 'chrome' is not defined.

(no-undef)